### DISCUSSION TITLE
⑂

### DISCUSSION THREAD

**Discussion by shnizzedy** [FCP-INDI/CPAC_regtest_pack#1] [General]
I [forked](https://github.com/sgiavasis/CPAC_regtest_pack/network/members) this repo from [sgiavasis/CPAC_regtest_pack](https://github.com/sgiavasis/CPAC_regtest_pack), but I wonder if it would be better to either transfer that repo to this org or to merge it into [FCP-INDI/CPAC-Development](https://github.com/FCP-INDI/CPAC-Development)?

(@FCP-INDI/developers )

**Comment by gkiar** [comment]
Thanks @shnizzedy ! I think we should transfer it, indeed. @sgiavasis, thoughts?
